### PR TITLE
bump up dependency

### PR DESCRIPTION
**Pull request by xiaoxichen**

### CODE REVIEW

**Comment by xiaoxichen** [comment]
@hkadayam  this is needed to allow us pining HS on 5.X in HO and unblock SM.

**Review by JacksonYao287 — APPROVED** [review]
since after  @hkadayam `s PR is merged , we need also do some change in homeobject to adopt the latest homestore changes, and before these homeobject changes are merged , homeobject and uppper layer will block on the building

so for the successfully building of current homeobject to enable nuobject 2.0 integration test, we can do this change for now.

**Pull request merged**
